src/main/java/edu/cornell/kfs/module/receiptProcessing/CsvBatchInputFileTypeBase.java

```diff
@@ -76,15 +79,14 @@ public Object parse(byte[] fileByteContent) throws ParseException {
 
         List<String> headerList = getCsvHeaderList();
         Object parsedContents;
-        try {
-            // validate csv header
-            ByteArrayInputStream validateFileContents = new ByteArrayInputStream(fileByteContent);
-            validateCSVFileInput(headerList, validateFileContents);
 
-            //use csv reader to parse the csv content
-            CSVReader csvReader = new CSVReader(
-                    new InputStreamReader(new ByteArrayInputStream(fileByteContent), StandardCharsets.UTF_8),
-                    ',', '"', '|');
+        validateCSVHeader(fileByteContent, headerList);
+
+        final CSVParser csvParser = new CSVParserBuilder().withSeparator(',').withQuoteChar('"').withEscapeChar('|').build();
+        try (
+                final InputStreamReader inputStreamReader = new InputStreamReader(new ByteArrayInputStream(fileByteContent), StandardCharsets.UTF_8);
+                final CSVReader csvReader = new CSVReaderBuilder(inputStreamReader).withSkipLines(1).withCSVParser(csvParser).build();
```

Comment:
This reader is being configured to skip a line, but a few lines down from here the existing code will automatically remove the first line in the list. Does that other line-deleting code need to be removed?